src/Masa.Blazor/Components/Sortable/MSortableProvider.cs

```diff
@@ -28,15 +28,14 @@ protected override void BuildRenderTree(RenderTreeBuilder builder)
     {
         builder.AddContent(0, sb =>
         {
-            var i = 0;
             foreach (var item in Items)
             {
                 Dictionary<string, object?> attrs = new()
                 {
                     { "data-id", ItemKey(item) }
                 };
                 var context = new SortableItemContext<TItem>(item, attrs);
-                sb.AddContent(i++, ItemContent?.Invoke(context));
+                sb.AddContent(1, ItemContent?.Invoke(context));
```

Comment:
It seems that the sequence in the for loop doesn't need to be incremented, as I checked the compiled source code of `@foreach` in Razor.